Redmine 25161 in the manage annual calendar view remove the number in the logo indicating the days with specificities

ManssourBlh opened the pull request:
https://dev.lutece.paris.fr/bugtracker/issues/25161

ryahiaoui commented:
Could you please make a clean pull request without merge commits and with the correct syntax?
"LUT-25109: message"

ManssourBlh commented:
> Could you please make a clean pull request without merge commits and with the correct syntax? "LUT-25109: message"

Hi @ryahiaoui, I've updated the pull request as requested. I've removed the merge commits. Additionally, I've amended the commit message to include the correct syntax with the issue key 'LUT-25161'. Please review the updated pull request at your convenience. Thank you!
